graphite/graphite.go

```diff
@@ -79,6 +84,37 @@ func (f *GraphitePublisher) Publish(metrics []plugin.Metric, cfg plugin.Config)
 		}
 		data := fmt.Sprintf("%v", m.Data)
 		logger.Debug("Metric ready to send %s:%s", key, data)
+
+		for _, char := range notAllowedChars {
```

Comment:
The metric namespace and tags are appended to key using a dot separator, which is why I'm checking for illegal characters in key.